test/parallel/test-buffer-readint16be.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+require('../common');
+const assert = require('assert');
+
+// testing basic functionality of readInt16BE()
+
+const buf = Buffer.from([42, 84, 168, 127]);
+const result = buf.readInt16BE(1);
+
+assert.strictEqual(result, 21672);
+
+assert.throws(
+  () => {
```

Comment:
@gibfahn IMHO the braces make it behave much more like a normal function, so it's more readable, especially when you're returning a value (`() => { return 123 }` vs `() => 123`).